semgrep-core/Core/AST/Bloom_filter.ml

```diff
@@ -15,29 +16,45 @@
 (*****************************************************************************)
 
 (* TODO *)
+type elt = string
+
 type t = {
   mutable added: bool;
+  filter: elt B.t;
 }
-[@@deriving show]
-
-type elt = string
 
 type bbool = No | Maybe
 
 (*****************************************************************************)
 (* API entry points *)
 (*****************************************************************************)
 
+let pp _formatter _bf = ()
+;;
+
 let create () =
-  { added = false }
+  { added = false;
+    filter = B.create ~error_rate:0.01 10 }
```

Comment:
maybe some quick explanation of what means the parameter and what is the intuition behind  those numbers?